ludwig/datasets/configs/av_mnist.yaml

```diff
@@ -0,0 +1,18 @@
+version: 1.0
+name: avmnist
+download_urls:
+  - http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz
+  - http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz
+  - http://yann.lecun.com/exdb/mnist/t10k-images-idx3-ubyte.gz
+  - http://yann.lecun.com/exdb/mnist/t10k-labels-idx1-ubyte.gz
```

Comment:
nit: Add comments to indicate that these two archive files are the test set files

Reply:
added